content/docs/alerting/configuration.md

```diff
@@ -462,10 +462,12 @@ endpoint:
   "externalURL": <string>,  // backling to the Alertmanager.
   "alerts": [
     {
+      "status": "<resolved|firing>",
       "labels": <object>,
       "annotations": <object>,
       "startsAt": "<rfc3339>",
-      "endsAt": "<rfc3339>"
+      "endsAt": "<rfc3339>",
+      "generatorURL": "string", // backling to the prometheus graph
```

Comment:
My point is that this comment is incorrect. GeneratorURL is a link to the process that generated the alert. It is not a link to a Prometheus graph.